jdaviz/components/plugin_switch.vue

```diff
@@ -28,7 +28,7 @@
 
 <script>
   module.exports = {
-    props: ['value', 'label', 'hint', 'api_hint', 'api_hints_enabled', 'use_eye_icon'],
+    props: ['value', 'label', 'hint', 'api_hint', 'api_hints_enabled', 'use_eye_icon', 'use_speaker_icon', 'use_icon'],
```

Comment:
looks like `use_speaker_icon` isn't actually used
```suggestion
    props: ['value', 'label', 'hint', 'api_hint', 'api_hints_enabled', 'use_icon'],
```